### PR TITLE
some implementations and fixes

### PR DESCRIPTION
**Pull request by Allvaa**
- implement pause option for ShoukakuPlayer#playTrack
- added connected field on PlayerUpdateEvent#state
- typings and typo fix

### CODE REVIEW

File: src/constants/ShoukakuConstants.js [hunk 1]
```diff
@@ -70,13 +70,15 @@ class ShoukakuConstants {
...
+    * @property {boolean} [pause=false] If `true`, the player will pause when the track starts playing.
     * @memberof ShoukakuConstants#
     */
     static get ShoukakuPlayOptions() {
         return {
             noReplace: true,
             startTime: undefined,
-            endTime: undefined
+            endTime: undefined,
+            pause: false
```
Comment (Deivu) [inline src/constants/ShoukakuConstants.js:81]: is the pause here optional on this new feature?

Comment (Allvaa) [reply src/constants/ShoukakuConstants.js:81]: yep, it's optional

Comment (Deivu) [reply src/constants/ShoukakuConstants.js:81]: just move the pause below noReplace, then update the docs to reflect the changes

**Comment by Deivu** [comment]
Seems good, thanks for the PR, merging it

**Pull request merged**